apps/src/code-studio/components/aiTutor/warningModal.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {useState} from 'react';
+import style from '@cdo/apps/aichat/views/chatWarningModal.module.scss';
```

Comment:
Totally on board for the plan to reference the aichat styling here 👍 Let's just be mindful that any future changes will show up in _both_ dialogs so we want to make sure that's desirable when the time comes for more UI/styling updates.

Reply:
What Erin said! You could even add a TODO in the code to migrate this in the future, if you think the reminder would be helpful for someone after you.

Reply:
Thanks! I'm going to leave the TODO for later when we get to more UI/styling updates.